docs/connectors/configuration-parameters/rate-limiting.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Rate Limiting"
+sidebar_position: 2
+---
+
+Destination connectors can be configured to limit the rate at which records can be written. This is especially useful when the destination resource has a rate limit to ensure that the connector does not exceed it. By default, Conduit does not limit the rate at which records are written.
+
+## Configuration parameters
+
+* `sdk.rate.perSecond`: Maximum times records can be written per second (0 means no rate limit).
+* `sdk.rate.burst`: Allow bursts of at most X writes (1 or less means that bursts are not allowed). Only takes effect if a rate limit per second is set.
```

Comment:
I was reading this and it made little sense to me, I checked the middleware and figured that it would make more sense to apply it _per record_ and not _per write_. The batch size shouldn't matter when configuring the rate limit IMO.

I opened this PR here, let's see what everyone thinks: https://github.com/ConduitIO/conduit-connector-sdk/pull/172

Reply:
@lovromazgon makes sense to me. I'll change it here